src/components/Settings/SettingsAddressbook.vue

```diff
@@ -22,14 +22,23 @@
   -->
 <template>
 	<div>
-		<li :class="{'disabled': !addressbook.enabled}" class="addressbook">
+		<li v-if="editingName" class="new-addressbook">
```

Comment:
so the input would appear in the popover menu and not replace the name as currently and in the old version? I can do it but need more guidance on how you would like it to look. It may make the popover menu a bit long?

Reply:
It will look like we do on the new Folder button on files 
![capture d ecran_2018-09-04_22-57-12](https://user-images.githubusercontent.com/14975046/45051701-1be60b00-b085-11e8-90d2-88ee3be5457a.png)

Reply:
ok I will change it to this or should I move that to a separate issue?

Reply:
Can be addressed in another pr I guess :)